tests/test_secrets.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# coding: utf-8
+"""
+Checks to make sure that our secrets checker, that runs prior to deployment, checks dev, integration, and staging...
+and doesn't check custom stage names.  Incorrect secrets on any of these 3 stages should throw a ValueError.
+"""
+import logging
+import sys
+import unittest
+import os
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from scripts.check_deployment_secrets import SecretsChecker
+from tests import needs_terraform
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestSecretCheck(unittest.TestCase):
+    @needs_terraform
+    def test_dev_secrets(self):
+        """Checks if the secrets in dev are gucci."""
+        s = SecretsChecker(stage='dev')
```

Comment:
Can `stage` be pulled in from `DSS_DEPLOYMENT_STAGE`? Then we would only need a single `test_secrets` method.

Reply:
Can do.

Reply:
Also a win of deleting my terrible comments.  XD